doc/clone-api.md

```diff
@@ -0,0 +1,35 @@
+[//]: # (cURLs in this file are generated from the .http file using http-client.env.json)
+
+# Clone API
+
+Let's say we have an account and a named subdomain. Because we're using [named subdomains](named-subdomains.md), the Gateway has a clone of the information in the (sub)domain.
```

Comment:
**concern:** I worry about calling this the "clone API", since it's specifically *not* accessing the data by cloning it. Let's at least be more explicit in this document that this is an *HTTP* API that allows you to interact with the remote clone, which is not actually using m-ld in the normal way, but rather a way to interact with the data from a client which can't or won't run its own clone.

Reply:
Haha good point, I'm so used to treating 'clone' as a noun! Will clarify as suggested.